app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
Okay, so it sounds like there's more accessibility work to be done.  Did you want me to put that stuff together in this PR?  Or can this be merged and a new PR be made for addressing the other issues?

It *sounds* like everybody agrees that "enter submits form" is better than "enter clicks nearest button", at least.